client/index.ts

```diff
@@ -1,5 +1,8 @@
-import validateConfig from '../packages/validate-config'
+import validateConfig from 'validate-config'
+import AutoSaveControlledForm, {
+  useAutoSaveControlledForm,
+} from './src/components/AutoSaveControlledForm'
 
 export { ReferenceDataStore } from './src/reference-data'
 export * as definitionService from './src/definition-manipulation'
-export { validateConfig }
+export { validateConfig, AutoSaveControlledForm, useAutoSaveControlledForm }
```

Comment:
While I'm sure this works, don't we only do this for exports outside the workspace? Since `useAutoSaveControlledForm` and `AutoSaveControlledForm` are both in client you can just export (so long as you don't export them as default in auto save tsx file)

Reply:
when you do `import {AutoSaveControlledForm} from 'toolkit-client'` it's going to look in index for something to import. 
Otherwise you have to do `import AutoSaveControlledForm from 'toolkit-client/dist/components/AutoSaveControlledForm'`